jira-clone/ToastStyles.js

```diff
@@ -0,0 +1,60 @@
+// Source: https://github.com/oldboyxx/jira_clone/blob/26a9e77b1789fef9cb43edb5d6018cf1663cf035/client/src/App/Toast/Styles.js#L1
+import styled from 'styled-components';
+
+import { color, font, mixin, zIndexValues } from 'shared/utils/styles';
+import { Icon } from 'shared/components';
+
+export const Container = styled.div`
+  z-index: ${zIndexValues.modal + 1};
+  position: fixed;
+  right: 30px;
+  top: 50px;
+`;
+
+export const StyledToast = styled.div`
+  position: relative;
+  margin-bottom: 5px;
+  width: 300px;
+  padding: 15px 20px;
+  border-radius: 3px;
+  color: #fff;
+  background: ${props => color[props.type]};
```

Comment:
> [!CAUTION]
> <ins>Defaults are a good idea</ins>: This is JavaScript, not TypeScript. One can't be sure:
> - what the structure of `props` is
> - that `props.type` is supplied by anyone using this component
> - that that someone supplied the right data type
> - that a string is given BUT it doesn't match any of the keys in `color`.
> Considering how many things can go wrong in the above, a better way to write this would have been:
> `background: ${({ type }) => type && color[type] ? color[type] : color.someDefault };